db/seeders/data/pending-applications.js

```diff
@@ -0,0 +1,449 @@
+'use strict';
+const faker = require('faker');
+const moment = require('moment');
+const uuidv4 = require('uuid/v4');
+
+const projectCountries = ['scotland', 'wales', 'england', 'northern-ireland'];
+const orgTypes = [
+    'unregistered-vco',
+    'unincorporated-registered-charity',
+    'charitable-incorporated-organisation',
+    'not-for-profit-company',
+    'school',
+    'college-or-university',
+    'statutory-body',
+    'faith-group'
+];
+
+const toDateParts = dt => {
+    return { day: dt.date(), month: dt.month() + 1, year: dt.year() };
+};
+
+const mockAddress = () => {
+    return {
+        line1: faker.address.streetAddress(),
+        townCity: faker.address.city(),
+        county: faker.address.county(),
+        postcode: 'B15 1TR'
+    };
+};
+
+module.exports = {
+    data: [
+        {
+            id: uuidv4(),
+            userId: 1,
+            formId: "awards-for-all",
+            applicationData: JSON.stringify({
```

Comment:
Would it be possible to sort the keys alphabetically in this object, to make it a bit easier to update in future?

Reply:
Also, could this just be a function which returns a blob of random data values each time, rather than duplicating all the field names each time?

eg. something like this:

```
const getRandomApplicationData = expiryDate => {
    return {
        projectName: faker.lorem.words(5),
        bankSortCode: faker.finance.account(6),
        ...,
        expiresAt: expiryDate
    };
};

const expiryDates = [
    moment().add('30', 'days'),
    moment().add('14', 'days'),
    moment().add('2', 'days'),
    moment(),
    moment().subtract('5', 'days')
];

module.exports = {
    data: expiryDates.map(expiryDate => {
        return {
            id: uuidv4(),
            userId: 1,
            formId: 'awards-for-all',
            applicationData: JSON.stringify(getRandomApplicationData(expiryDate.toDate()))
        };
    })
};


```

Reply:
Makes complete sense! @davidrapson mentioned the same as well. Will work on this change now.